apps/web/src/components/Day.tsx

```diff
@@ -126,7 +126,7 @@ export const DayContent = (props: DayContentProps) => {
   return (
     <ReactSortable
       id={day.date}
-      className="mt-4 flex flex-auto flex-col overflow-y-auto overflow-x-hidden"
```

Comment:
I forgot to specify but just doing this introduces a regression:
![2023-07-06 20 12 19](https://github.com/richardguerre/flow/assets/53370763/df3d4921-ba18-453a-b2ad-c92c0dcd1029)

It should look like this:
![2023-07-06 20 13 18](https://github.com/richardguerre/flow/assets/53370763/1ad767a9-7ff7-4352-897f-c0344b74b4ad)

Reply:
You can either remove these lines and fix that regression, or keep the lines and find a way to still show the cards when dragging between day columns.

Not sure, but you can look into the `Lists` component (in `Lists.tsx`) which is the component rendered on the right side of the IndexView and potentially using `flex-1` like it does (I remember having a similar overflow issue and that one tailwind class fixed it for some reason).